docs/ref/models/querysets.txt

```diff
@@ -2230,13 +2230,19 @@ instances, generally with one query::
     >>> objs[0].headline = 'This is entry 1'
     >>> objs[1].headline = 'This is entry 2'
     >>> Entry.objects.bulk_update(objs, ['headline'])
+    2
+
+.. versionchanged:: 4.0
+
+    This function earlier returned ``None``. Now, it returns the count of the
+     objects updated.
```

Comment:
```suggestion
    The return value with the number of objects updated was added.
```

Reply:
Although i am not a native english speaker, i took the liberty to change this to `Return value with the number of updated objects was added` as first `the` appeared unnecessary to me, and `updated objects` as per the above suggestion on the release notes. Let me know if you think this isn't appropriate.